sormas-api/src/main/java/de/symeda/sormas/api/user/UserRight.java

```diff
@@ -628,6 +628,9 @@
 			SURVEILLANCE_OFFICER,
 			EVENT_OFFICER
 	),
+	EVENTPARTICIPANT_DELETE(
+			ADMIN
```

Comment:
Just saw that we also allow the NATIONAL_USER to delete events, so that user should probably also be allowed to delete participants.